tests/testthat/test-rstudio_markers.R

```diff
@@ -94,14 +94,12 @@ test_that("rstudio_source_markers apply to print within rstudio", {
   empty <- withr::local_tempfile()
   file.create(empty)
 
-  with_mock(
-    `rstudioapi::hasFun` = function(x, ...) TRUE,
-    `rstudioapi::callFun` = function(...) cat("matched\n"), {
-    l <- lint(tmp, seq_linter())
-    expect_output(print(l), "matched", fixed = TRUE)
+  mockery::stub(print.lints, "rstudioapi::hasFun", function(x, ...) TRUE)
+  mockery::stub(print.lints, "rstudio_source_markers", function(x) cat("matched\n"))
```

Comment:
I couldn't get `mockery::stub(rstudio_source_markers, ...)` calls to do the correct thing but figured we can resort to testing that `rstudio_source_markers()` is called here.

Reply:
add that as a comment, maybe with a TODO you we think it's preferable to eventually test `rstudio_source_markers()` instead?

Reply:
I like the idea of unit testing `rstudio_source_markers()` separately, but I'm not sure what exactly to test other than that the calls made via `rstudioapi` fire and get the same input across time.